player/models.py

```diff
@@ -222,6 +228,20 @@ def toggle_shuffle(self, to_active):
             self.save()
             self.send_update_message()
 
+    def order_playlist(self, id, action):
+        print("action : " + action + " on : " + id)
+        if action == "top":
```

Comment:
if action no in ['top', 'up', 'down', 'bot]:
    return
playlist = PlaylistTrack.objects.get(room=self, track__music_id=id)
getattr(playlist, action)()